package.json

```diff
@@ -69,14 +69,14 @@
   },
   "lint-staged": {
     "./**/*.{js,jsx,ts,tsx}": [
-      "npm run lint"
+      "yarn run lint"
     ],
     "./**/*.{js,jsx,ts,tsx,scss}": [
-      "npm run format"
+      "yarn run format"
     ]
   },
   "volta": {
     "node": "14.19.1",
-    "npm": "6.14.16"
+    "yarn": "1.22.19"
```

Comment:
We should start using the latest LTS node version. Could you create a new issue for it and track?

Reply:
Sure.